src/modules/rppt_tensor_geometric_augmentations.cpp

```diff
@@ -634,6 +668,26 @@ RppStatus rppt_crop_mirror_normalize_gpu(RppPtr_t srcPtr,
                                               roiType,
                                               rpp::deref(rppHandle));
     }
+    else if ((srcDescPtr->dataType == RpptDataType::U8) && (dstDescPtr->dataType == RpptDataType::F32))
```

Comment:
Keep the same order of calls for host and gpu, either is fine.

Reply:
Done